primary.tf

```diff
@@ -0,0 +1,68 @@
+resource "aws_s3_bucket" "primary" {
+  provider = aws.primary
+
+  bucket = var.primary_name
+
+  force_destroy = var.force_destroy
+
+  tags = var.tags
+}
+
+resource "aws_s3_bucket_versioning" "primary" {
```

Comment:
I'd recommend to mention explicitly in the README that it's not possible to set `aws_s3_bucket_versioning` on the created buckets outside of the module, as the module's versioning configuration is required for the replication to work.

Reply:
see 36c7e6283f4f00471fb4213f29341d1c659b66df